CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed command `scanapi spec-file.yaml` to `scanapi run spec-file.yaml`. [#247](https://github.com/scanapi/scanapi/pull/247)
 - Moved Documentation from README.md to the website. [#250](https://github.com/scanapi/scanapi/pull/250)
 - Local and global configuration. [#254](https://github.com/scanapi/scanapi/pull/254)
+- Show absolute path to the report in CLI. [#277](https://github.com/scanapi/scanapi/pull/277)
```

Comment:
Hey, we use the Unreleased section to store the changes for the next release. That means this change line should be under  `Unreleased`. 

We have created a wiki page to help newcomers to understand how to add a new entry on changelog since it is not intuitive: https://github.com/scanapi/scanapi/wiki/Changelog

Reply:
Thanks for watching this